tst/EnergyPlus/unit/AirflowNetworkHVAC.unit.cc

```diff
@@ -19752,6 +19759,60 @@ TEST_F(EnergyPlusFixture, AirflowNetwork_ZoneOrderTest)
         "    CommaAndHTML,            !- Column Separator",
         "    InchPound;               !- Unit Conversion",
 
+        "  NodeList,",
+        "  Zone Exhaust Node_unit1 Nodes,",
+        "  Zone3DehumidifierInlet,",
+        "  Zone Exhaust Node_unit1;",
+
+        "  ZoneHVAC:Dehumidifier:DX,",
+        "    North Zone Dehumidifier, !- Name",
+        "    always_avail,            !- Availability Schedule Name",
+        "    Zone3DehumidifierInlet,  !- Air Inlet Node Name",
+        "    Dehumidifier Outlet Node,!- Air Outlet Node Name",
+        "    50.16,                   !- Rated Water Removal {L/day}",
+        "    3.412,                   !- Rated Energy Factor {L/kWh}",
+        "    0.12036,                 !- Rated Air Flow Rate {m3/s}",
+        "    ZoneDehumidWaterRemoval, !- Water Removal Curve Name",
+        "    ZoneDehumidEnergyFactor, !- Energy Factor Curve Name",
+        "    ZoneDehumidPLFFPLR,      !- Part Load Fraction Correlation Curve Name",
+        "    10.0,                    !- Minimum Dry-Bulb Temperature for Dehumidifier Operation {C}",
+        "    32.0,                    !- Maximum Dry-Bulb Temperature for Dehumidifier Operation {C}",
+        "    0.0;                     !- Off-Cycle Parasitic Electric Load {W}",
```

Comment:
Add a dehumidifier to this test to create the same situation as the defect file (multiple exhaust nodes). The defect file uses a VRF terminal.

Reply:
Nice job just modifying an existing test here.